### DISCUSSION TITLE
How to programmatically use brush to draw along a path?

### DISCUSSION THREAD

**Discussion by lanyusan** [fabricjs/fabric.js#9669] [Q&A] [answered]
My use case is to use a certain brush to draw along a programmatically defined path. How to do it by code? 

I haven't found any code examples.

Thanks.

**Comment by ShaMan123** [comment]
You mean you want to animate a path rendering?

**Reply by lanyusan** [reply]
What I meant is that I don't want to set canvas to `drawingMode` and let Fabirc take care of brush drawing when user interacts with canvas with mouse. 

Instead I have already got a path, let's say it is a beautify curve drawn by Picasso. I want to use a certain brush to paint alone the curve without having to ask user to interactively click canvas along the curve.

How do I do that?

**Reply by ShaMan123** [reply, marked as answer]
> You mean you want to animate a path rendering?

The answer is yes
Refer to #9077 

However, I would suggest using `bezier-js` instead
